### PR TITLE
ci: coverage reports

### PR DESCRIPTION
**Pull request by tpluscode**
I set this up similarly to b59 so that we run each package's tests and collect coverage individually

### CODE REVIEW

**Comment by tpluscode** [comment]
@ludovicm67 please add [CODECOV_TOKEN secret](https://app.codecov.io/gh/zazuko/trifid/settings)

File: packages/ckan/test/ckan.test.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ describe('@zazuko/trifid-plugin-ckan', () => {
     trifidListener.close()
   })
 
-  describe('basic tests', () => {
+  describe.skip('basic tests', () => {
```
Comment (ludovicm67) [inline packages/ckan/test/ckan.test.js:56]: Why are they skipped?

Comment (tpluscode) [reply packages/ckan/test/ckan.test.js:56]: Ah, my mistake, obviously

**Comment by ludovicm67** [comment]
> @ludovicm67 please add [CODECOV_TOKEN secret](https://app.codecov.io/gh/zazuko/trifid/settings)

Done just now :)

**Review by ludovicm67 — APPROVED** [review]

**Comment by ludovicm67** [comment]
This will require some good rebase I would say :)

**Comment by tpluscode** [comment]
I don't know why not all coverage reports are correctly consumed but we probably have to live with it. Re https://github.com/codecov/feedback/issues/276

**Comment by ludovicm67** [comment]
I guess we can live with it.
It also seems that some files are not reported in the coverage reports.

**Comment by tpluscode** [comment]
> I guess we can live with it. It also seems that some files are not reported in the coverage reports.

For example?

**Comment by ludovicm67** [comment]
The core package, there is only the `lib` directory that gets reported: https://app.codecov.io/github/zazuko/trifid/tree/cov/packages%2Fcore

**Comment by tpluscode** [comment]
Good catch. Was missing `--all` in that one script

**Comment by ludovicm67** [comment]
Great, thanks!
I will merge it :)

**Pull request merged**